count.js

```diff
@@ -0,0 +1,9 @@
+function count(a){
+   var array = a.split('');
+   var i = 0;
+   while (typeof array[i] === 'string') { 
+      i++;
+   }
+   return(i);       
+}
```

Comment:
Строки (`String`) итерируемые (массивоподобные) по умолчанию, т.е. нет необходимости дополнительно разделять их через `split()`, т.к. вы получаете тот же массив

Также, можно опустить проверку в условии цикла, на то что очередной элемент массива является типом `String`, т.к. при достижении конца массива `a[i]` будет возвращать `undefined`, в следствии чего, результат вычисления условия цикла будет ложным и цикл прервется

```suggestion
function count(a){
   var i = 0;
   while (a[i]) { 
      i++;
   }
   return(i);       
}
```